package.json

```diff
@@ -133,6 +133,7 @@
     "phonegap-plugin-push": "git+https://github.com/moodlemobile/phonegap-plugin-push.git#moodle-v3",
     "promise.prototype.finally": "3.1.0",
     "rxjs": "5.5.12",
+    "sjcl": "1.0.8",
```

Comment:
I think this is fine because it's all local. sjcl is quite large compared to other libraries.